actions/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use scorecard's official Docker image as base.
+ARG DOCKER_SHA
+FROM gcr.io/openssf/scorecard@sha256:${DOCKER_SHA} as base
+
+# Build our image and update the root certs.
+FROM debian:9.5-slim
```

Comment:
Why not build from distroless?

Reply:
+1

Reply:
AFAIK, we cannot use a shell in distroless. The entry point is a shell. For distrless, we'll need to new build `scorecard-action` as a golang app. If you don't mind, I'll create an issue for tracking and we can do that in a follow-up PR once everything else works.
Please let me know if you disagree.

Reply:
So I think it's better to have a wrapper, se https://github.com/ossf/scorecard/pull/1069#discussion_r716982443
Long-term we can create a golang app to avoid using a shell and then we'll be able to move to distroless. Can do that in follow-up PR once the action flow works. I'll create an issue for tracking. 
Let me know if you disagree.

Reply:
Works for me. I think we shouldn’t use the Docker container for GitHub action. As part of that we can solve this issue. Thanks

Reply:
Could you add an action metadata file in this PR - https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action#creating-an-action-metadata-file?

I'm ok with changing the internals in follow-up PRs, if we can define the API that will be exposed to users in this PR.

Reply:
this is the API:
```
uses: docker://scorecard-action
        with:
          policy_file: .github/scorecard/policy.yml
          sarif_file: results.sarif
          repo_token: ${{ secrets.GITHUB_TOKEN }}
```

The `action.yml` is not used unless it defines a `composite` action, AFAIK. I have it defined but did not upload it because I found it nt to be necessary. Shall I upload it in this PR?

Reply:
> Works for me. I think we shouldn’t use the Docker container for GitHub action. As part of that we can solve this issue. Thanks

You mean https://docs.github.com/en/actions/creating-actions/creating-a-composite-action?
Can do that, yes.

Reply:
create https://github.com/ossf/scorecard/issues/1072 for tracking

Reply:
I wasn't suggesting a `composite action`. I just mean let's add the `action.yml` file even though it won't be used since it clarifies the API.